src/policy/fees.cpp

```diff
@@ -907,8 +909,9 @@ bool CBlockPolicyEstimator::Read(CAutoFile& filein)
         LOCK(m_cs_fee_estimator);
         int nVersionRequired, nVersionThatWrote;
         filein >> nVersionRequired >> nVersionThatWrote;
-        if (nVersionRequired > CLIENT_VERSION)
-            return error("CBlockPolicyEstimator::Read(): up-version (%d) fee estimate file", nVersionRequired);
+        if (nVersionRequired > CLIENT_VERSION) {
+            throw std::runtime_error(strprintf("up-version (%d) fee estimate file", nVersionRequired));
```

Comment:
It looks like this could change behaviour (in addition to changing logging), but perhaps the exception handling is such that only logging is affected?

Reply:
> perhaps the exception handling is such that only logging is affected?

Yes, the exception handling will only log, but this is clear from reading the code. Maybe I am misunderstanding your question?

Reply:
Oh, when expanding the diff that is clearly so :)  I should have expanded the diff before commenting.